src/cachier/core.py

```diff
@@ -260,7 +259,12 @@ def func_wrapper(*args, **kwds):
             if _allow_none or entry.value is not None:
                 _print("Cached result found.")
                 now = datetime.now()
-                if now - entry.time <= _stale_after:
+                stale_delta = (
+                    min(_stale_after, max_age)
+                    if max_age is not None
+                    else _stale_after
+                )
```

Comment:
I wish we could have longer lines like standard 120, so this would be single line :)